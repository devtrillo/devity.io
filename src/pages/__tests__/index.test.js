import { render } from '@testing-library/react';
import HomePage from '../index';
import React from 'react';

describe('The homePage', function () {
  it('should not change', function () {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
