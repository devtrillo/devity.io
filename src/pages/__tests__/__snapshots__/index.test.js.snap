// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The homePage should not change 1`] = `
<div>
  <div>
    Hello world!
    <a
      href="/animated"
    >
      ANIMATED
    </a>
  </div>
</div>
`;
